umn/source/faqs/buckets_and_objects/lifecyle_policy_as_best_practice_to_delete_objects_and_to_deal_with_versioning.rst

```diff
@@ -0,0 +1,49 @@
+.. _obs_faq_0199:
+
+Lifecyle Policy as Best Practice to Delete Objects and to Deal with Versioning
```

Comment:
as this is FAQ should be stylized as question

Reply:
Fixed